src/Plugin/Block/EuFooterBlock.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\oe_corporate_blocks\Plugin\Block;
+
+use Drupal\Component\Utility\NestedArray;
+use Drupal\Core\Cache\CacheableMetadata;
+use Drupal\Core\Plugin\ContainerFactoryPluginInterface;
+
+/**
+ * Provides the corporate EU footer block.
+ *
+ * @Block(
+ *   id = "oe_eu_footer",
```

Comment:
Same here, please change into:
```
oe_corporate_blocks_eu_footer
```

Also change this in the demo module configuration.